### DISCUSSION TITLE
no person detected

### DISCUSSION THREAD

**Discussion by Holger79** [nextcloud/recognize#773] [Q&A] [answered]
Hi,

i´ve the problem, that no persons are detected. I run a Nextcloud (26.0.0) the last Docker image on my Synology NAS with the recognize app at Version 3.7.0  and Tensorflow in the WASM mode.

I like to test the app, so I created a new Docker container an put more then 75 pictures of my self in it. The app tells that its working:
![grafik](https://user-images.githubusercontent.com/13599559/227739909-c01eddc8-eb92-4261-9c6f-d10ed0cd7db2.png)

but there are no people detected.

If I run occ recognize:cluster-faces command it got the folowing output:

Clustering face detections for user test
ClusterDebug: Retrieving face detections for user test
ClusterDebug: Not enough face detections found

In the Database all oc_recognize_* tables are empty only in the oc_recognize_face_detections  table has 93 records

Does anyone have an idea what I do wrong / what I´ve to to that the app detect people?

**Comment by Holger79** [comment, marked as answer]
I found a solution, I add more then 50 pictures with other people and now it works

**Reply by marcelklehr** [reply]
Yep,

> Not enough face detections found

This is the key information. I'll try to make this more obvious
